script.js

```diff
@@ -1,4 +1,100 @@
+// setting base
+var numOfPlayers = 2;
+var numOfDice = 2;
+var currentPlayer = 1;
+var playerOneScore;
+var playerTwoScore;
+
+//// game modes
+// var GAME_MODE_DICE_ROLL = "game mode dice roll";
+// var GAME_MODE_DICE_ORDER = "game mode dice order";
+
+// var GAME_MODE_DICE_ROLL;
+// var GAME_MODE_DICE_ORDER;
+
+// current game mode
+var gameMode = "dice roll";
+
+// dice rolling
+var diceRolls = [];
+var diceOneRoll = Math.floor(Math.random() * 6 + 1);
+var diceTwoRoll = Math.floor(Math.random() * 6 + 1);
+var diceThreeRoll = Math.floor(Math.random() * 6 + 1);
+var diceFourRoll = Math.floor(Math.random() * 6 + 1);
+// var diceOne = [];
+// var diceTwo = [];
+// var getDiceRolls = function () {
+//   Math.floor(Math.random() * 6 + 1);
+// };
+// var diceOneRoll = getDiceRolls();
+// var diceTwoRoll = getDiceRolls();
+// console.log(`first comp dice ${diceOneRoll}, second comp dice ${diceTwoRoll}`);
+// main
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var output = "";
+
+  if (gameMode == "dice roll") {
+    if (currentPlayer == 1) {
+      console.log(`current player is ${currentPlayer}`);
+      output = `Hi Player ${currentPlayer}!<br><br>Here are your dice rolls:
+    <br>Dice 1: ${diceOneRoll}
+    <br> Dice 2: ${diceTwoRoll}
+    <br><br>Please select the order of the dice. Which dice will be your first order?`;
+      gameMode = "dice order";
+    } else if (currentPlayer == 2) {
+      console.log(`current player is ${currentPlayer}`);
+      output = `Hi Player ${currentPlayer}!<br><br>Here are your dice rolls:
+    <br>Dice 1: ${diceThreeRoll}
+    <br> Dice 2: ${diceFourRoll}
+    <br><br>Please select the order of the dice. Which dice will be your first order?`;
```

Comment:
these lines of code are similar, consider putting them in a function to avoid repetition of code